index.html

```diff
@@ -1,38 +1,320 @@
+            </div>
+            <button class="card-button">See Project</button>
+          </div>
+        </div>
+        <div class="card">
+          <div class="card-body">
+            <img
+              src="./images/photo-1533035353720-f1c6a75cd8ab.jpg"
+              alt=""
+              class="card-image"
+            />
+            <h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+            <div>
+              <ul class="card-tags">
+                <li>Ruby on Rails</li>
+                <li>css</li>
+                <li>JavaScript</li>
+                <li>html</li>
+              </ul>
+            </div>
+            <button class="card-button">See Project</button>
+          </div>
+        </div>
+        <div class="card">
+          <div class="card-body">
+            <img
+              src="./images/photo-1533035353720-f1c6a75cd8ab.jpg"
+              alt=""
+              class="card-image"
+            />
+            <h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+            <div>
+              <ul class="card-tags">
+                <li>Ruby on Rails</li>
+                <li>css</li>
+                <li>JavaScript</li>
+                <li>html</li>
+              </ul>
+            </div>
+            <button class="card-button">See Project</button>
+          </div>
+        </div>
+        <div class="card">
+          <div class="card-body">
+            <img
+              src="./images/photo-1533035353720-f1c6a75cd8ab.jpg"
+              alt=""
+              class="card-image"
+            />
+            <h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+            <div>
+              <ul class="card-tags">
+                <li>Ruby on Rails</li>
+                <li>css</li>
+                <li>JavaScript</li>
+                <li>html</li>
+              </ul>
+            </div>
+            <button class="card-button">See Project</button>
+          </div>
+        </div>
+        <div class="card">
+          <div class="card-body">
+            <img
+              src="./images/photo-1533035353720-f1c6a75cd8ab.jpg"
+              alt=""
+              class="card-image"
+            />
+            <h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+            <div>
+              <ul class="card-tags">
+                <li>Ruby on Rails</li>
+                <li>css</li>
+                <li>JavaScript</li>
+                <li>html</li>
+              </ul>
+            </div>
+            <button class="card-button">See Project</button>
+          </div>
+        </div>
+      </section>
+    </main>
+      <section class="about-us">
+        <h1 class="about-heading">About Me</h1>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, do not hestiate to contact me.
+        </p>
+      </section>
```

Comment:
- [ ] As suggested by the previous reviewer, Kindly use the `section tag` is supposed to grab from the About up to the end of the skills part. Make sure you remove all the other `section` tag and replace them even with some generic `div`

Reply:
All section tags in About us section is removed and only one section tag covering frim About right up the end of skills